### PR TITLE
Add get all admins

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Gateway/ProfilesFacade.fs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿module ProfilesFacade
```
Comment (jebbinSCL) [inline Gateway/ProfilesFacade.fs]: Shall this be ProfilesFacade or ProfilesAndHandlesFacade?

Comment (jebbinSCL) [inline Gateway/ProfilesFacade.fs]: Since there is already ProfilesProxy that is. I feel like having a profilesProxy and Facade is confusing.

Comment (dmorrisSL) [inline Gateway/ProfilesFacade.fs]: My feeling is that it's best to describe a façade by how it looks from the outside - if it (A) passes back both profiles and handles (as in handles on their own, not within a profile), then call it ProfilesAndHandles, whereas if it (B) passes back only profiles (i.e handles only get returned 'inside' a profile), then ProfilesFacade sounds better. 

Looking at the code I think it's (B), but my F# isn't great.

Comment (dmorrisSL) [inline Gateway/ProfilesFacade.fs]: In any case, I'll say :shipit:

Comment (jebbinSCL) [inline Gateway/ProfilesFacade.fs]: Ah I was thinking of it the other way around. I was thinking of describing it by what it provides access to. I was thinking that although the overall result is a combined result, it has to access handles and profiles to get it. 
I'll leave it as ProfilesFacade for now then.

Comment (jebbinSCL) [inline Gateway/ProfilesFacade.fs]: Ah yeah, ProfilesFacade seems right after a bit more reading.

**Comment by dmorrisSL** [comment]
:shipit:

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**